deploy/ocm-agent-operator-managedfleetnotifications/oidc-deleted-limited-support.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     name: oidc-deleted-notification
     summary: Cluster is in Limited Support due to unsupported cloud provider configuration
     notificationMessage: |-
-      Your cluster requires you to take action because Red Hat is not able to access the infrastructure with the provided credentials. Please restore the credentials and permissions provided during install. If you have deleted the associated OpenIDConnectProvider, please recreate it by executing the command: rosa create oidc-provider --mode manual --cluster $CLUSTER
+      Your cluster requires action because Red Hat is either unable to access the infrastructure with the provided credentials or the credentials lack the necessary permissions to perform required actions. Please restore the credentials and permissions configured during the installation. If the associated OpenIDConnectProvider has been deleted, you can recreate it by running the following command: rosa create oidc-provider --cluster $CLUSTER. If etcd encryption is enabled, ensure that Red Hat has access to your custom KMS key. For more details, refer to the documentation: https://docs.openshift.com/rosa/rosa_hcp/rosa-hcp-creating-cluster-with-aws-kms-key.html#creating-cluster-with-aws-kms-key. 
```

Comment:
the `Red Hat has access to your custom KMS key` sounds a bit weird to me

Reply:
Agreed, thoughts on: `ensure that Red Hat can perform required actions on your custom KMS key"?

Reply:
yeah, this sounds better :+1: